common/src/main/java/com/wynntils/mc/event/ScreenInitEvent.java

```diff
@@ -26,4 +26,17 @@ public Screen getScreen() {
     public boolean isFirstInit() {
         return firstInit;
     }
+
+    public static class Pre extends ScreenInitEvent {
+        public Pre(Screen screen, boolean firstInit) {
+            super(screen, firstInit);
+        }
+    }
+
+    // FIXME: This event does not trigger on first init. See the section near ScreenMixin.java L61
```

Comment:
I don't understand why this is really an issue. We already have a "Pre" mixin, can't you just copy it, and shift it "after" the target?

Reply:
I did try to use the @return of the same method, but it has the same effect as the Pre mixin. The screen's coordinates are still the ones from before the re-init.